clarity/src/vm/ast/definition_sorter/tests.rs

```diff
@@ -32,24 +33,41 @@ use crate::vm::ClarityVersion;
 
 #[template]
 #[rstest]
-#[case(ClarityVersion::Clarity1)]
-#[case(ClarityVersion::Clarity2)]
-fn test_clarity_versions_definition_sorter(#[case] version: ClarityVersion) {}
+#[case(ClarityVersion::Clarity1, StacksEpochId::Epoch2_05)]
+#[case(ClarityVersion::Clarity1, StacksEpochId::Epoch21)]
+#[case(ClarityVersion::Clarity2, StacksEpochId::Epoch21)]
+#[case(ClarityVersion::Clarity2, StacksEpochId::Epoch22)]
```

Comment:
will Clarity1 be allowed in Epoch 2.2? if so, would that need to be an additional case?

Reply:
Yes, 2.2 should support Clarity1, but the fix is gated on the epoch only, not the Clarity version. It can't hurt to include that combo any way.

Reply:
Addressed in 06e546951.